File: 2635/2635.py
while True:
    try:
        n = int(input())
        data = []
        for _ in range(n):
            data.append(input())
        q = int(input())
        for _ in range(q):
            text = input()
            ans = 0
            max_len = 0
            for word in data:
                if word.startswith(text):
                    ans += 1
                    max_len = max(max_len, len(word))
            if ans == 0:
                print(-1)
            else:
                print(ans, max_len)
    except EOFError:
        break

File: 2635/2635.js
const lines = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');

let index = 0;

while (true) {
    const n = parseInt(lines[index++]);
    if (isNaN(n)) break;

    const data = [];
    for (let i = 0; i < n; i++) {
        data.push(lines[index++]);
    }

    const q = parseInt(lines[index++]);
    for (let j = 0; j < q; j++) {
        const text = lines[index++];
        let ans = 0;
        let maxLen = 0;
        for (const word of data) {
            if (word.startsWith(text)) {
                ans++;
                maxLen = Math.max(maxLen, word.length);
            }
        }
        if (ans === 0) {
            console.log(-1);
        } else {
            console.log(ans, maxLen);
        }
    }
}